File: Kaitoss.Web/Models/About.cs
using System.ComponentModel.DataAnnotations;

namespace Kaitoss.Web.Models
{
    public class About
    {
        public int Id { get; set; }
        [Display(Name = "Title")]
        [Required(ErrorMessage = "Title is required")]
        public string? Title { get; set; }

        [Display(Name = "Who we are")]
        [Required(ErrorMessage = "Who we are is required")]
        public string? WhoWeAre { get; set; }
  
        [Display(Name = "Vision")]
        [Required(ErrorMessage = "Vision is required")]
        public string? Vision { get; set; }

        [Display(Name = "History")]
        [Required(ErrorMessage = "History is required")]
        public string? History { get; set; }
        public string? ImageUrl { get; set; }

    }
}

File: Kaitoss.Web/wwwroot/Components/js/About.js
var dataTable;

$(document).ready(function () {
    LoadDataTable();
});

function LoadDataTable() {
    dataTable = $('#tblData').DataTable({
        "ajax": {
            "url": "/About/GetAll"
        },
        "columns": [
            {
                "data": "imageUrl", "name": "imageUrl", "defaultContent": "<i>-</i>",
                "render": function (data, type, row, meta) {

                    return '<img src="' + data + '" class="img-datatable" />';
                }, "width": "5%"
            },
            { "data": "title", "width": "5%" },
            { "data": "whoWeAre", "width": "5%" },
            { "data": "vision", "width": "5%" },
            { "data": "history", "width": "5%" },
            {
                "data": "id",
                "render": function (data) {
                    return `
                <a class="btn-primary btn" href="./About/Upsert?id=${data}"><i class="bi bi-pencil-square"></i></a>
                <a class="btn-danger btn" onClick="Delete('./About/DeleteItem/${data}')" ><i class="bi bi-trash3"></i></a>
                           `
                },
                "width": " 5%"
            }
        ]
    });
}

function Delete(url) {
    Swal.fire({
        title: 'Are you sure?',
        text: "You won't be able to revert this!",
        icon: 'warning',
        showCancelButton: true,
        confirmButtonColor: '#3085d6',
        cancelButtonColor: '#d33',
        confirmButtonText: 'Yes, delete it!'
    }).then((result) => {
        if (result.isConfirmed) {
            $.ajax({
                url: url,
                type: 'DELETE',
                success: function (data) {
                    if (data.success) {
                        dataTable.ajax.reload();
                        toastr.success(data.message);
                    } else {
                        toastr.error(data.message);
                    }
                }
            })
        }
    })
}